docs/source/x_ext.rst

```diff
@@ -454,18 +482,20 @@ Issue interface
   +------------------------+----------------------------------------+-----------------------------------------------------------------------------------------------------------------+
   | ``mode``               | :ref:`mode_t <mode>`                   | Effective privilege level, as used for load and store instructions.                                             |
   +------------------------+----------------------------------------+-----------------------------------------------------------------------------------------------------------------+
+  | ``hartid``             | :ref:`hartid_t <hartid>`               | Identification of the hart offloading the instruction.                                                          |
+  +------------------------+----------------------------------------+-----------------------------------------------------------------------------------------------------------------+
   | ``id``                 | :ref:`id_t <id>`                       | Identification of the offloaded instruction.                                                                    |
   |                        |                                        |                                                                                                                 |
   |                        |                                        |                                                                                                                 |
   +------------------------+----------------------------------------+-----------------------------------------------------------------------------------------------------------------+
 
-An issue request transaction is defined as the combination of all ``issue_req`` signals during which ``issue_valid`` is 1 and the ``id`` remains unchanged.
+An issue request transaction is defined as the combination of all ``issue_req`` signals during which ``issue_valid`` is 1 and the ``hartid`` remains unchanged.
 A |processor| is allowed to retract its issue request transaction before it is accepted with ``issue_ready`` = 1 and it can do so in the following ways:
-
+ 
 * Set ``issue_valid`` = 0.
-* Keep ``issue_valid`` = 1, but change the ``id`` signal (and if desired change the other signals in ``issue_req``).
+* Keep ``issue_valid`` = 1, but change the ``hartid`` signal (and if desired change the other signals in ``issue_req``).
```

Comment:
change the ``hartid`` signal -> change the ``hartid`` and/or ``id`` signal

Reply:
fixed in 08f84d1, commits squashed for merge into main